lib/github_changelog_generator.rb

```diff
@@ -139,7 +139,7 @@ def fetch_merged_at_pull_requests
       }
 
       if @options[:verbose]
-        puts 'Fetching merged dates... Done!'
+        puts "Fetching merged dates: Done!"
```

Comment:
Thanks for cleanup! I think there is should be single quotes, since you don't use any parameters.